api/PclusterApiHandler.py

```diff
@@ -561,7 +561,7 @@ def get_identity():
 
     access_token = request.cookies.get("accessToken")
     id_token = request.cookies.get("idToken")
-    if not access_token:
+    if not (access_token and id_token):
```

Comment:
Please follow this PR up with a test case for this. You can look at `test_pcluster_api_handler.py` and `test_authenticate.py` to see how cookies have been handled/mocked in tests

Reply:
I will, thanks!